libcontainer/configs/mount_unsupported.go

```diff
@@ -0,0 +1,34 @@
+//go:build !linux
+// +build !linux
```

Comment:
Why unsupported instead of unix?

Reply:
I'm following existing pattern in the package (ie, see cgroups_unsupported.go)

Reply:
Oh, makes sense. Thanks!